docs/pages/components/calendar.md

```diff
@@ -10,755 +10,1217 @@ summary:
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">Aug.</span>
-						</td>
-					</tr>
-					<tr class="fd-calendar__row">
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">Sep.</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">Oct.</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">Nov.</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">Dec.</span>
-						</td>
-					</tr>
-				</tbody>
-			</table>
-		</div>
-		<div class="fd-calendar__years is-hidden" aria-hidden="true" id="FTXrD591">
-			<table class="fd-calendar__table" role="grid">
-				<tbody class="fd-calendar__group">
-					<tr class="fd-calendar__row">
-						<td class="fd-calendar__item fd-calendar__item--current">
-							<span class="fd-calendar__text" role="button">2018</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2019</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2020</span>
-						</td>
-						<td class="fd-calendar__item is-selected">
-							<span class="fd-calendar__text" role="button" aria-selected="true">2021</span>
-						</td>
-					</tr>
-					<tr class="fd-calendar__row">
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2022</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2023</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2024</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2025</span>
-						</td>
-					</tr>
-					<tr class="fd-calendar__row">
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2026</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2027</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2028</span>
-						</td>
-						<td class="fd-calendar__item">
-							<span class="fd-calendar__text" role="button">2029</span>
-						</td>
-					</tr>
-				</tbody>
-			</table>
-		</div>
-	</div>
+    <header class="fd-calendar__header">
+        <div class="fd-calendar__navigation">
+			<div class="fd-calendar__action fd-calendar__action--arrow-left">
+    			<button type="button" class="fd-button fd-button--light sap-icon--slim-arrow-left" aria-label="Previous"></button>
+    		</div>
+    		<div class="fd-calendar__action">
+    			<button type="button" class="fd-button fd-button--light" aria-selected="false" aria-expanded="false" aria-controls="fm3cr569">January</button>
```

Comment:
Let's remove useless `aria` labels,  clicking on it causes 
![image](https://user-images.githubusercontent.com/26483208/75427906-9d9bb980-5947-11ea-8c15-c7741cc8415e.png)
error.

Reply:
The `aria-controls` seems to be the problematic property here. It doesn't appear to correctly link to any valid id on the page, either.